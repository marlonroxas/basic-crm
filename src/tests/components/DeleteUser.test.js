import React from 'react'
import { shallow } from 'enzyme'

import DeleteUser from '../../components/DeleteUser/index'

test('Should render DeleteUser correctly', () => {
  const wrapper = shallow(<DeleteUser />)
  expect(wrapper).toMatchSnapshot()
})