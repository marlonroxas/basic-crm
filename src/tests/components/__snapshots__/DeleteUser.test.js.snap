// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render DeleteUser correctly 1`] = `
<form
  onSubmit={[Function]}
>
  <div
    className="form-group"
  >
    Are you sure you want to delete 
    ?
  </div>
  <div
    className="form-group form-group--actions"
  >
    <button
      className="primary-btn"
    >
      Delete
    </button>
    <button
      className="cancel-btn"
      onClick={[Function]}
    >
      Cancel
    </button>
  </div>
</form>
`;
